raiwidgets/tests/common_utils.py

```diff
@@ -12,6 +14,9 @@ def check_success_criteria(self, flask_server_prediction_output):
         assert WidgetRequestResponseConstants.error not in \
             flask_server_prediction_output
 
+        json.dumps(flask_server_prediction_output[
```

Comment:
Is this test util file or actual file? Asking since it is under _tests/_ folder

Reply:
Yes, this is in the test folder just doing the validation that the output of the dashboard is json serialized.